lib/fastly-rails/action_controller/cache_control_headers.rb

```diff
@@ -7,12 +7,14 @@ module CacheControlHeaders
     # Defaults are:
     #  Cache-Control: 'public, no-cache'
     #  Surrogate-Control: 'max-age: 30 days
-    # custom config example: 
+    # custom config example:
     #  {cache_control: 'public, no-cache, maxage=xyz', surrogate_control: 'max-age: blah'}
     def set_cache_control_headers(max_age = FastlyRails.configuration.max_age, opts = {})
-      request.session_options[:skip] = true    # no cookies
-      response.headers['Cache-Control'] = opts[:cache_control] || "public, no-cache"
-      response.headers['Surrogate-Control'] = opts[:surrogate_control] || "max-age=#{max_age}"
+      unless ENV["FASTLY_CACHE_DISABLED"].present?
```

Comment:
I would prefer we didn't duplicate this logic across the lib. If we changed the name of this env var down the line, it would suck to have to change it multiple places in code.